src/lib/dns-server/clients/dns_server.rb

```diff
@@ -0,0 +1,72 @@
+# encoding: utf-8
+
+# File:	clients/dns-server.ycp
+# Package:	Configuration of dns-server
+# Summary:	Main file
+# Authors:	Jiri Srain <jsrain@suse.cz>
+#		Lukas Ocilka <locilka@suse.cz>
+#
+# $Id$
+#
+# Main file for dns-server configuration. Uses all other files.
+module Yast
+  class DnsServerClient < Client
+    include Yast::Logger
+
+    Yast.import "DnsServer"
+    Yast.import "PackageSystem"
+
+    def main
+
+      #**
+      # <h3>Configuration of the dns-server</h3>
+
+      textdomain "dns-server"
+
+      # The main ()
+      log.info("----------------------------------------")
+      log.info("DnsServer module started")
+
+      # main ui function
+      @ret = nil
```

Comment:
NP: We do not need to store the result, simple `ret` is sufficient.

And it would be assigned later, this line can be actually removed...